t/plugin/limit-count.t

```diff
@@ -1178,7 +1178,7 @@ passed
 
 
 
-=== TEST 37: plugin is bound to the service
+=== TEST 37: plugin is bound to the service and upstream is the domain name
```

Comment:
I think we can also add route and bind service_id in this one.

Reply:
updated.